src/transform.js

```diff
@@ -1,11 +1,29 @@
 // external dependencies
 import compose from 'lodash/fp/compose';
+import isEqual from 'lodash/isEqual';
 import isUndefined from 'lodash/isUndefined';
+import moize from 'moize';
+import autoprefixer from 'autoprefixer';
+import postcss from 'postcss';
 
 // constants
-import {
-  PREFIXER
-} from './constants';
+import {DEFAULT_AUTOPREFIXER_OPTS} from './constants';
+
+/**
+ * @function getAutoPrefixer
+ *
+ * @description
+ * Function that returns an autoprefixer instance configured with the provided options.
+ * Memoized for better performance (uses lodash isEqual for equality check)
+ *
+ * @param {object} options
+ * @returns {object}
+ */
+export const getAutoPrefixer = moize((autoPrefixerOpts) => {
+  return postcss([
+    autoprefixer(autoPrefixerOpts)
+  ]);
+}, {equals: isEqual});
```

Comment:
is there a reason you need to do a deep equality check on the options? it makes the library size substantially larger, when it seems like using the standard strict equality would be sufficient.

Reply:
Using standard strict equality for `moize` will result in cache misses when an equal but new autoprefixerOptions object is provided to any style tag. I wanted to reduce unnecessary initialization of the autoprefixer object but if you feel that adding the `isEqual` for deep equality check is a deal breaker, I can change it back

Reply:
e.g.

```
<Style autoprefixerOptions={{ remove: true }}>{cssText}</Style>

// later in the code
<Style autoprefixerOptions={{ remove: true }}>{cssText}</Style>
```

will lead to 2 autoprefixer instances being created

Reply:
I hear what you're saying, but that sounds like something that should be called out in documentation, not pre-optimized to combat against.

Reply:
yep, makes sense, I'll update the PR and add a note to the README 👍